pkg/decision/composite_feature_service.go

```diff
@@ -44,19 +44,16 @@ func NewCompositeFeatureService(compositeExperimentService ExperimentService) *C
 
 // GetDecision returns a decision for the given feature and user context
 func (f CompositeFeatureService) GetDecision(decisionContext FeatureDecisionContext, userContext entities.UserContext) (FeatureDecision, error) {
-	lastElementIndex := len(f.featureServices) - 1
-	for index, featureDecisionService := range f.featureServices {
-		featureDecision, err := featureDecisionService.GetDecision(decisionContext, userContext)
+	var featureDecision = FeatureDecision{}
+	var err = errors.New("")
```

Comment:
This should just be `var err error` so that it is initialized to nil. Otherwise we always return a non-nil error

Reply:
This was on purpose because if there were no feature services, `var err error` would return nil error, are we ok with it?

Reply:
Yeah that is fine

Reply:
done, thanks 👍